zh_cn/about/license.txt

```diff
@@ -0,0 +1,33 @@
+Ruby 是受版权保护的自由软件，著作权人为 Yukihiro Matsumoto <matz@netlab.jp>.
+根据2-clause BSDL协议 (请参阅 BSDL 文件)和下述条款，你将被授权对它进行重新分发和（或）修改:
+
+  1. 你可以制作和（或）分发源代码格式的完整副本而不受限制，但必须包含原有的一切版权声明及免责声明。
+
+  2. 在满足下述条文中至少一项的前提下，你可以以任何方式修改你的副本:
+
+       a) 将您的修改部分发布于公共领域（或以其他形式确保公众能够免费得到修改后的副本），
+       包括（但不限于）在新闻组（或其他类似媒介）中张贴修改部分，或授权Ruby作者引用修改部分。
```

Comment:
Also in my opinion, Ruby should add a space before & after:

```
或授权 Ruby 作者引用修改部分。
```

reads slightly better.